project.godot

```diff
@@ -246,7 +246,6 @@ translations=PoolStringArray( "res://locale/en.po", "res://locale/fr.po", "res:/
 [logging]
 
 file_logging/enable_file_logging=true
-file_logging/log_path="user://logs/log.txt"
```

Comment:
This change should be reverted

Reply:
I got this line on 249 is that i've to revert?

file_logging/max_log_files=8

Reply:
The diff shows that this entire line:
```
file_logging/log_path="user://logs/log.txt"
```
has been deleted...

Reply:
You made this part worse, now you have deleted 2 lines.

Red lines starting with a - in a diff shows what has been removed:
![Kuvakaappaus - 2021-01-25 14-06-47](https://user-images.githubusercontent.com/3796411/105703923-a493c900-5f16-11eb-8c46-f3182818aa67.png)


This PR should not have any changes to project.godot before this can be accepted